src/lib/game.ts

```diff
@@ -87,14 +87,16 @@ export function generateEmojiArt(
   const medals = guessModeMedal(guessMode);
   const count = gameLost(solution, rows) ? "X" : rows.length;
   let result = `${name}${medals} ${gameDay + 1} ${count}/${ROW_COUNT}\n`;
-  for (let i = 0; i < rows.length; ++i) {
-    const row = rows[i];
-    for (const x of findRowStates(solution, row)) {
+  for (const [i, row] of rows.entries()) {
```

Comment:
Nice, didn't even know Array had a method for that

Reply:
Yeah I didn't either, I was looking for `.enumerate()` lol